packages/fx-core/tests/common/deps-checker/cases/dotnet.it.ts

```diff
@@ -46,15 +49,15 @@ describe("DotnetChecker E2E Test - first run", async () => {
       new TestTelemetry()
     );
 
-    const isInstalled = await dotnetChecker.isInstalled();
-    assert.isFalse(isInstalled, ".NET is not installed, but isInstalled() return true");
-
-    const depsInfo: DepsInfo = await dotnetChecker.getDepsInfo();
+    const depsInfo = await dotnetChecker.getInstallationInfo();
     assert.isNotNull(depsInfo);
-    assert.isFalse(depsInfo.isLinuxSupported, "Linux should not support .NET");
+    assert.isFalse(depsInfo.isInstalled, ".NET is not installed, but isInstalled() return true");
+    assert.isFalse(depsInfo.details.isLinuxSupported, "Linux should not support .NET");
 
-    const res = await dotnetChecker.resolve();
-    assert.isTrue(res.isOk() && res.value);
+    const spyChecker = sandbox.spy(dotnetChecker);
+    const res = await spyChecker.resolve();
+    assert.isTrue(res.isInstalled);
+    assert.isTrue(spyChecker.getInstallationInfo.calledTwice);
```

Comment:
Why called twice? Is this fix to reduce to only one call?

Reply:
If it is the first time installation, will check whether the installation is success.

Reply:
make sense